app/controllers/base_trainee_controller.rb

```diff
@@ -92,6 +96,13 @@ def filtered_trainees
     )
   end
 
+  def unfiltered_trainees
```

Comment:
I wonder if rather than `unfiltered_trainees` we could call this `total_trainees` ? Only a very small point, I just found `unfiltered_trainees` a bit less clear

Reply:
I considered the name total_trainees but i thought it might conflict with the total_trainees_count method above (which i renamed in this pr from total_trainees)